### PR TITLE
ui v2: buttons

### PR DESCRIPTION
**Pull request by dschaller**
### Description

New designs for buttons. Some colors are inlined into the component here and will be extracted later when we are updating global themes of the application. A subsequent PR will be opened to add icon support.

**Primary**
Screen Shot 2020-11-13 at 3 03 41 PM

**Primary with mouseover**
Screen Shot 2020-11-13 at 3 03 45 PM

**Primary disabled**
Screen Shot 2020-11-13 at 3 03 45 PM

**Destructive | Danger**
Screen Shot 2020-11-13 at 3 03 49 PM

**Destructive | Danger with mouseover**
Screen Shot 2020-11-13 at 3 03 52 PM

**Panel**
Screen Shot 2020-11-13 at 3 03 37 PM

**Panel with neutral mouseover**
Screen Shot 2020-11-13 at 3 03 31 PM

### Testing Performed

manual with storybook and local app

### CODE REVIEW

File: frontend/packages/core/src/AppProvider/themes.tsx [hunk 1]
```diff
@@ -88,10 +89,12 @@ const Theme: React.FC<ThemeProps> = ({ children }) => {
   const theme = lightTheme;
   return (
     <MuiThemeProvider theme={theme()}>
-      <StyledThemeProvider theme={theme()}>
-        <CssBaseline />
-        <StylesProvider injectFirst>{children}</StylesProvider>
-      </StyledThemeProvider>
+      <EmotionThemeProvider theme={theme()}>
+        <StyledThemeProvider theme={theme()}>
```
Comment (dschaller) [inline frontend/packages/core/src/AppProvider/themes.tsx:93]: 😭 until we can remove styled from core we will have 3 theming frameworks

File: frontend/packages/core/src/stories/button.stories.tsx [hunk 2]
```diff
@@ -14,13 +14,25 @@ export default {
...
-  text: "continue",
+export const Primary = Template.bind({});
+Primary.args = {
+  text: "Continue",
 };
 
 export const Destructive = Template.bind({});
 Destructive.args = {
-  text: "delete",
-  destructive: true,
```
Comment (scarlettperry) [inline frontend/packages/core/src/stories/button.stories.tsx:25]: should we call this one "Caution" too, like in button group?

File: frontend/packages/core/src/stories/button-group.stories.tsx [hunk 3]
```diff
@@ -25,13 +28,13 @@ Default.args = {
   ],
 };
 
-export const Destructive = Template.bind({});
-Destructive.args = {
```
Comment (scarlettperry) [inline frontend/packages/core/src/stories/button-group.stories.tsx:29]: (couldn't comment on lines where changes weren't made) for "Default" - should we update it to "Primary", like in the button story?

**Comment by danielhochman** [comment]
comparing figma and storybook and noticing some differences

image

File: frontend/packages/core/src/stories/button.stories.tsx (hunk 2 above)
Comment (dschaller) [reply frontend/packages/core/src/stories/button.stories.tsx:25]: Reverted back to destructive

**Comment by danielhochman** [comment]
looks better, can we ship the roboto font change to UIV2 first so we can be sure we're 1:1?

File: frontend/packages/core/src/button.tsx [hunk 4]
```diff
@@ -1,47 +1,73 @@
...
-  extends Pick<
-    MuiButtonProps,
-    "disabled" | "endIcon" | "onClick" | "size" | "startIcon" | "type" | "variant"
-  > {
+const OutlinedButton = styled(StyledButton)({
+  border: "1px solid #0D1030",
+});
+
+export interface ButtonProps
+  extends Pick<MuiButtonProps, "disabled" | "endIcon" | "onClick" | "size" | "startIcon" | "type"> {
```
Comment (danielhochman) [inline frontend/packages/core/src/button.tsx:49]: we can drop the size prop i think

File: api/api/v1/annotations.proto [hunk 5]
```diff
@@ -20,4 +20,5 @@ extend google.protobuf.MethodOptions {
   // Use a random high number that won't conflict with annotations from other
   // libraries.
   Action action = 58901;
+  bool disable_audit = 58902;
```
Comment (danielhochman) [inline api/api/v1/annotations.proto:23]: bad merge, you merged main and not uiv2

**Comment by dschaller** [comment]
Rebased off UIV2 with Roboto and using 700 font-weight:

Screen Shot 2020-11-16 at 11 24 26 AM

**Comment by danielhochman** [comment]
from what i can tell in figma, neutral hover should be #eff0f2 but it's #cccccc

**Review by danielhochman — APPROVED** [review]
:tada:

**Pull request merged**